src/WarcraftLegacies.Source/Mechanics/Fel Horde/FelHordeJuggernautDeath.cs

```diff
@@ -0,0 +1,39 @@
+﻿using MacroTools;
+using MacroTools.Wrappers;
+using System;
+using WCSharp.Shared.Data;
+using static War3Api.Common;
+
+namespace WarcraftLegacies.Source.Mechanics.Fel_Horde
+{
+
+ 
+  internal class FelHordeJuggernautDeath
```

Comment:
I think we should rename this to just `JuggernautDeath`. It's already in the Fel Horde folder so the specificity is redundant.